linuxNew/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec

```diff
@@ -45,9 +48,19 @@
 %global vers_arch2 ppc64le
 %global vers_arch3 s390x
 %global vers_arch4 aarch64
+%global vers_arch5 arm
 %global src_num 6
 %global sha_src_num 7
 %endif
+%ifarch armv7l
```

Comment:
On my system 32-bit arm is more than this. I'd suggest to use `%{arm}` instead here.

```
$ rpm --eval '%{arm}'
armv3l armv4b armv4l armv4tl armv5tl armv5tel armv5tejl armv6l armv6hl armv7l armv7hl armv7hnl armv8l armv8hl armv8hnl armv8hcnl
```

Reply:
Did you mean `%ifarch arm`? Yes, with fedora 33 and 34 `$ rpm --eval '%{arm}'` got same result as yours. However with rpm build --target arm --rebuild I got the following errors.  Details see https://github.com/adoptium/installer/issues/390#issuecomment-979489941

```
RPM build errors:
xargs: invalid number "%{_smp_build_ncpus}" for -P option
Try 'xargs --help' for more information.
+ /usr/lib/rpm/redhat/brp-ldconfig
+ /usr/lib/rpm/brp-compress
+ /usr/lib/rpm/brp-strip-comment-note /usr/bin/strip /usr/bin/objdump
+ /usr/lib/rpm/redhat/brp-strip-lto /usr/bin/strip
xargs: invalid number "%{_smp_build_ncpus}" for -P option
Try 'xargs --help' for more information.
error: Bad exit status from /var/tmp/rpm-tmp.lPSnk9 (%install)``

Reply:
I meant: instead of `%ifarch armv7l` write `%ifarch %{arm}`

Reply:
Thanks! Updated.